.gitignore

```diff
@@ -3,3 +3,4 @@ composer.lock
 docs
 vendor
 tests/temp
+.idea/
```

Comment:
This line belongs in [a global gitignore](https://murze.be/2014/12/create-a-global-gitignore/). Could you remove it?